examples/classifier.c

```diff
@@ -205,13 +205,37 @@ void train_classifier_valid(char *datacfg, char *cfgfile, char *weightfile, int
     int classes = option_find_int(options, "classes", 2);
 
 
-    char log_file_path[1024];
+    char time_stamp_dir[1024], log_file_path[1024], summary_file_path[1024];
+
+    sprintf(time_stamp_dir, "%s/%s", backup_directory, full_time_stamp());
+    mkdir(time_stamp_dir, S_IRWXU | S_IRWXG | S_IRWXO);
+
+    sprintf(log_file_path, "%s/%s", time_stamp_dir, params.log_file);
+    FILE* log_file_ts = fopen(log_file_path, "w+");
+
+    sprintf(summary_file_path, "%s/%s", time_stamp_dir, params.hyper_param_file);
+    FILE* summary_file_ts = fopen(summary_file_path, "w+");
 
     sprintf(log_file_path, "%s/%s", backup_directory, params.log_file);
     FILE* log_file = fopen(log_file_path, weightfile ? "a+" : "w+");
 
-    sprintf(log_file_path, "%s/%s", backup_directory, params.hyper_param_file);
-    FILE* summary_file = fopen(log_file_path, weightfile ? "a+" : "w+");
+    sprintf(summary_file_path, "%s/%s", backup_directory, params.hyper_param_file);
+    FILE* summary_file = fopen(summary_file_path, "w+");
```

Comment:
Is there a reason to have two summary files placed in distinct places?

Reply:
The first summary file (summary_file_ts) points to the new directory, created with time stamp, and saves all the information about the current run. The second points to the current running summary, so we always have fresh info about the training execution, even if it stops somehow.